scripts/zones/Monastic_Cavern/Zone.lua

```diff
@@ -8,8 +8,15 @@ require('scripts/globals/treasure')
 local zoneObject = {}
 
 zoneObject.onInitialize = function(zone)
-    UpdateNMSpawnPoint(ID.mob.ORCISH_OVERLORD)
-    GetMobByID(ID.mob.ORCISH_OVERLORD):setRespawnTime(math.random(900, 10800))
+    -- NM Persistence
+    local timeOfDeath          = GetServerVariable("[POP]Overlord_Bakgodek")
+    local kills                = GetServerVariable("[PH]Overlord_Bakgodek")
+    local popNow      = math.random(1, 5) == 3 or kills > 6
```

Comment:
What is going on here with formatting?

Reply:
ah in vs code it's formatted equally, I'll undo this change.